### ISSUE TITLE
getSchema is not available on `maybe` types

### ISSUE THREAD

**Issue by justinbmeyer** [canjs/can-type#32]
For `can-query-logic` and `can-crud` to work, `canReflect.getSchema()` needs to return an `Or` type like:

```js
canReflect.getSchema( type.maybe( String ) ) //-> 
{
  type: "Or",
  values: [null, undefined, String]  
}
```

A similar thing should work for any convert type too:

```js
canReflect.getSchema( type.convert( type.maybe( String ) ) ) //-> 
{
  type: "Or",
  values: [null, undefined, String]  
}
```

**Comment by justinbmeyer** [comment]
It might be a problem on other maybe types too: String, Number, etc

**Comment by matthewp** [comment]
Should be fixed now in `can-type@1.0.0-pre.6`
